<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SelectSkyboxDialog</class>
 <widget class="QDialog" name="SelectSkyboxDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>559</width>
    <height>363</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="1" column="0">
      <widget class="QPushButton" name="xButton">
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>100</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>100</width>
         <height>100</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../../EditorResources.qrc">
         <normaloff>:/icons/Icons/sb_x.png</normaloff>:/icons/Icons/sb_x.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>97</width>
         <height>97</height>
        </size>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="4">
      <widget class="QPushButton" name="nyButton">
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>100</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>100</width>
         <height>100</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../../EditorResources.qrc">
         <normaloff>:/icons/Icons/sb_ny.png</normaloff>:/icons/Icons/sb_ny.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>97</width>
         <height>97</height>
        </size>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QPushButton" name="yButton">
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>100</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>100</width>
         <height>100</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../../EditorResources.qrc">
         <normaloff>:/icons/Icons/sb_y.png</normaloff>:/icons/Icons/sb_y.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>97</width>
         <height>97</height>
        </size>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="3">
      <widget class="QPushButton" name="nxButton">
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>100</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>100</width>
         <height>100</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../../EditorResources.qrc">
         <normaloff>:/icons/Icons/sb_nx.png</normaloff>:/icons/Icons/sb_nx.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>97</width>
         <height>97</height>
        </size>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QPushButton" name="zButton">
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>100</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>100</width>
         <height>100</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../../EditorResources.qrc">
         <normaloff>:/icons/Icons/sb_z.png</normaloff>:/icons/Icons/sb_z.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>97</width>
         <height>97</height>
        </size>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QPushButton" name="nzButton">
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>100</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>100</width>
         <height>100</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../../EditorResources.qrc">
         <normaloff>:/icons/Icons/sb_nz.png</normaloff>:/icons/Icons/sb_nz.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>97</width>
         <height>97</height>
        </size>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Expanding</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="cancelButton">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="okButton">
       <property name="text">
        <string>Ok</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../EditorResources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>cancelButton</sender>
   <signal>clicked()</signal>
   <receiver>SelectSkyboxDialog</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>441</x>
     <y>341</y>
    </hint>
    <hint type="destinationlabel">
     <x>881</x>
     <y>468</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
